packages/nuxt/package.json

```diff
@@ -42,7 +42,7 @@
     "nuxt": "3.x"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.12.2",
```

Comment:
Do you know, is it normal for plugins like sentry to directly depend on nuxt kit or is it more like a peer dep thing?

Reply:
I saw it as a dependency in other Nuxt modules and therefore added it there too